transportation-data-publishing/data_tracker/signal_pms_fulcrum.py

```diff
@@ -42,6 +42,12 @@ def get_postgre_records():
     params = {}
 
     postgre_records = pgrest.select(params=params)
+    postgre_records_df = pd.DataFrame.from_dict(postgre_records)
+
+    # temporary fix to remove duplicate pm records
```

Comment:
I like having some comments for explanation. I'm wondering, what would be the more permanent fix? Would that be worth adding as a comment?

Reply:
If we are going to use fulcrum for an extended time. I'm going to rewrite some part of this script. Currently, this script blends dictionaries/dataframe operations together which makes it hard to maintain.